packages/database/src/server/getDbMigrator.js

```diff
@@ -51,8 +93,26 @@ export const getDbMigrator = (forCli = false) =>
         },
       },
       cmdOptions: {
-        'migrations-dir': path.join(__dirname, '../core/migrations'),
+        'migrations-dir': SERVER_MIGRATION_DIR,
       },
     },
     forCli ? cliCallback : appCallback,
   );
+
+  // No need to exclude non-server migrations if we're creating a new migration
+  if (!process.argv.includes('create')) {
+    // 'core/migrations' folder is shared between server and browser
+    // We need to exclude non-server migrations before they are run
+    // ie: excluding migrations that have _meta.targets.includes('browser')
+    // This hook is called BEFORE the migrations are run,
+    // so we temporarily remove non-server migrations before they are run
+    instance.registerAPIHook(() => {
+      removeDirectoryIfExists(SERVER_MIGRATION_DIR);
+      createDirectory(SERVER_MIGRATION_DIR);
+      copyDirectory(MIGRATIONS_DIR, SERVER_MIGRATION_DIR);
+      removeNonServerMigrations();
+    });
+  }
```

Comment:
@edmofro This change is about allowing some migrations to be run on server only!

I've attempted to replace db-migrate with the new custom MigrationManager. However I've realized quite late that there were some legacy migrations that use a lot of built in features of db-migrate such as `db.createTable`, `db.addColumn`, etc... So I figured it's actually cleaner if we keep using db-migrate, and just manually remove the non server migration files like this. Let me know what you think 🙏

Reply:
Yeah happy with that